index.md

```diff
@@ -36,53 +38,89 @@
   </a>
 </p>
 
-## About
-How many deployments have you done that needed to be rolled back due to a missing character in a configuration file in your repo? If you're like most teams that number is greater than zero. The config file validator was created to solve this problem by searching through your project and validating the syntax of all configuration files. 
-
-### Where can you use this tool?
-* In a CI/CD pipeline as a quality gate
-* On your desktop to validate configuration files as you write them
-* As a library within your existing go code
-
-### What types of files are supported?
+## Supported config files formats:
 * XML
 * JSON
 * YAML
 * TOML
 * INI
-* Properties
 * HCL
-* PLIST
 * CSV
+* Properties
+* Apple PList XML
```

Comment:
Consider alphabetizing the list in all instances of this list.

Reply:
Great idea! Will do